docker-compose.yml

```diff
@@ -207,6 +210,19 @@ services:
       interval: 10s
       timeout: 10s
       retries: 30
+  garage:
+    image: dxflrs/garage:v1.0.1
+    volumes:
+      - type: bind
+        read_only: true
+        source: ./garage.toml
+        target: /etc/garage.toml
+      - "sentry-garage:/var/lib/garage/"
+    healthcheck:
+      test: "wget -no-verbose --tries=1 --spider http://localhost:3903/health || exit 1"
+      interval: 20s
```

Comment:
This seems rather long, is this waiting for any other service to come up first? Would it be useful to include a depends_on block here?

Reply:
Nope, we can definitely lower it. Right now it doesn't even work so I'll revert it for now.